verification/src/solidity/verifier.rs

```diff
@@ -1 +1,99 @@
+#![allow(dead_code, unused)]
```

Comment:
I'm afraid I would be needed to constantly add and remove those directives in that case. I'll remove it when the `Verifier` being implemented so that no dead or unused code will be in the final version